test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -727,6 +772,91 @@ public void CreateDisplayMetadata_IsFlagsEnum_ReflectsModelType(Type type, bool
                 .ThenBy(item => item.Key.Name, StringComparer.Ordinal));
         }
 
+        private DataAnnotationsMetadataProvider CreateLocalizingProvider()
+        {
+            var stringLocalizer = new Mock<IStringLocalizer>(MockBehavior.Strict);
+            stringLocalizer
+                .Setup(loc => loc["LOC_Two"])
+                .Returns<string>((k => new LocalizedString(k, $"{k} {CultureInfo.CurrentCulture}")));
```

Comment:
When mocking things this way, why is `DataAnnotationsMetadataProviderTest.EnumWithLocalizedDisplayNames.resx` needed? I prefer this approach over testing the implementation details of the string localizer.